### DISCUSSION TITLE
Need docs of SocketsHttpHandler.PlaintextStreamFilter

### DISCUSSION THREAD

**Discussion by GF-Huang** [dotnet/runtime#53323] [Q&A] [answered]
It's no any descriptions or usage now.

![image](https://user-images.githubusercontent.com/4510984/119744969-c2bb0700-bebf-11eb-99eb-79539b9945f3.png)

**Comment by huoyaoyuan** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/5644458/120095543-2875f000-c159-11eb-9cfa-ac574a95cd66.png)
This may be a docs tooling issue. It has doc in the source.
